### DISCUSSION TITLE
JanetRS - Janet high-level bindings for Rust: version 0.5.0

### DISCUSSION THREAD

**Discussion by GrayJack** [janet-lang/janet#888] [Show and tell]
I just released a new version of JanetRS. This one is a smaller one due to my new time constrains.

In this release we have these highlighted features:

- Add `catch` arg to [`janet_fn`](https://docs.rs/janetrs/latest/janetrs/attr.janet_fn.html) attribute macro that adds code to catch Rust panics and transform them into Janet panics
- Fix `tuple::{IntoIter, Iter}::size_hint` implementation
- Migrate to Rust 2021

The full changelog can be found [here](https://github.com/GrayJack/janetrs/blob/v0.5.0/CHANGELOG.md)

**Comment by andrewchambers** [comment]
Thanks for working on this GrayJack :)
